src/plugins/intel_cpu/src/nodes/topk.cpp

```diff
@@ -1985,11 +1986,11 @@ void TopK::initSupportedPrimitiveDescriptors() {
                                                            ov::element::u8};
 
     ov::element::Type dataPrecision = getOriginalOutputPrecisionAtPort(TOPK_DATA);
-    if (dataPrecision == ov::element::bf16 && !mayiuse(avx512_core)) {
-        THROW_CPU_NODE_ERR("gets incorrect isa for BF16! AVX512 must be supported!");
-    }
     bool precisionSupported = std::find(std::begin(supportedPrecision), std::end(supportedPrecision), dataPrecision) !=
                               std::end(supportedPrecision);
+    // BF16 is not supported for AVX2_VNNI_2 platforms
+    precisionSupported =
+        (dataPrecision == ov::element::bf16 && mayiuse(cpu::x64::avx2_vnni_2)) ? false : precisionSupported;
```

Comment:
It means systems with pure avx2 or sse isa support bf16?

Reply:
changed to non avx512_core limitation